### DISCUSSION TITLE
multi color line chart

### DISCUSSION THREAD

**Discussion by sky06walker** [ecomfe/vue-echarts#657] [Q&A] [answered]
Hi, I need below chart in vue with typescript but I not able to find any related sample, can anyone guide me on this. Thanks

https://whchong.tinytake.com/msc/NzQxNzE4Ml8yMDMwODYwMw

**Comment by ntnyq** [comment]
Maybe line chart with gradient.

**Comment by sky06walker** [comment, marked as answer]
end up I'm using echart to fulfill my requirement.
